src/data/common_textures.inc.c

```diff
@@ -2566,92 +2566,92 @@ f32 D_0D009808[][4] = {
 };
 
 // Odometer speedometer
-s8 D_0D009958[] = {
-    #include "textures/common/132B50_09958.i4.inc.c"
+s8 gTextureSpeedometer[] = {
```

Comment:
Teeeechnically Odometer... But I think most people would use speedometer so we'll stick with that.

Reply:
The odometer measures the total distance the vehicle has traveled.

Reply:
I'd like to retract my previous statement.